pyerrors/correlators.py

```diff
@@ -1190,11 +1187,11 @@ def __rmatmul__(self, y):
         else:
             return NotImplemented
 
-    def __truediv__(self, y):
+    def __truediv__(self, y: Union[Corr, float, ndarray, int]) -> "Corr":
```

Comment:
`Obs` and `CObs` are missing here. BTW: does that just mean that we don't call the function with these types in our tests?

Reply:
Yes, that's my understanding. monkeytype derived these type hints from our tests.